.circleci/config.yml

```diff
@@ -34,7 +34,7 @@ executors:
 
   builder_pg_geth:
```

Comment:
I saw you bumped geth to 1.9.15 in https://github.com/omisego-images/docker-elixir-omg/pull/45/files#diff-093788b95b8bce4248ae369334936680088f911fc96463fc973308a091cceaacR48.
will you bump in this repo?